samples/document-snippets/row.js

```diff
@@ -14,7 +14,7 @@
  */
 
 const snippets = {
-  createRow: (instanceId, tableId) => {
+  createRow: async (instanceId, tableId) => {
```

Comment:
lines  52, 53:  If the targeted cell is unset, it will be treated as a containing an
     // empty string.	 Please rephrase it to "If the targeted cell is unset, it will be treated as a cell containing an empty string."  Search and change everywhere.